src/main/java/codeu/model/data/Message.java

```diff
@@ -63,6 +66,17 @@ public String getContent() {
     return content;
   }
 
+  public String getStyledContent(){
```

Comment:
I don't think you need to break this up into two methods. I'd just have one method that is getStyledContent with the code from below.